addons/mail/static/src/models/messaging_menu/messaging_menu.js

```diff
@@ -81,5 +82,10 @@ registerModel({
         isOpen: attr({
             default: false,
         }),
+        mobileMessagingNavbar: one2one('mail.mobile_messaging_navbar', {
+            default: insertAndReplace(),
+            inverse: 'messagingMenu',
+            isCausal: true,
+        }),
```

Comment:
I would be expecting no default, and only create when in mobile.

Also should probably be suffixed `View`